axelrod/strategies/darwin.py

```diff
@@ -44,16 +43,18 @@ class Darwin(Player):
     valid_callers = ["play"]    # What functions may invoke our strategy.
 
     def __init__(self) -> None:
-        super().__init__()
+        self.outcomes = None
         self.response = Darwin.genome[0]
+        super().__init__()
 
     def receive_match_attributes(self):
         self.outcomes = self.match_attributes["game"].scores
 
     def strategy(self, opponent: Player) -> Action:
         # Frustrate psychics and ensure that simulated rounds
         # do not influence genome.
-        if inspect.stack()[1][3] not in Darwin.valid_callers:
```

Comment:
oops.  didn't notice that i changed this.  i had been trying to understand inspect.stack().  Change it back, or no need.